public/app/plugins/datasource/loki/components/LokiContextUi.tsx

```diff
@@ -163,6 +181,31 @@ export function LokiContextUi(props: LokiContextUiProps) {
 
   return (
     <div className={styles.wrapper}>
+      <Button
+        title="Revert to initial log context query."
```

Comment:
I think we should remove that, because that will fiddle with the tooltip:
![image](https://github.com/grafana/grafana/assets/8092184/d9dcd775-73b1-47b9-8646-5e07de3bcbee)

```suggestion
```

Reply:
Fixed in https://github.com/grafana/grafana/pull/68484/commits/50414156b6896ba392672a4c216f07c791dccfb9